napari/_qt/qt_main_window.py

```diff
@@ -336,6 +350,19 @@ def _add_help_menu(self):
         )
         self.help_menu.addAction(about_key_bindings)
 
+    def _toggle_axes(self, action):
+        text = action.text()
+        if text == 'None':
+            self.qt_viewer.viewer.axes_visible = False
+        elif text == 'Colored':
+            self.qt_viewer.viewer.axes_visible = True
+            self.qt_viewer.viewer.axes_style = 'Colored'
```

Comment:
Is there a common convention about which color means which axis? To be honest, even if there's no common convention, we should probably consider matching what Imaris does. I imagine a reasonable chunk of our users might have previous experience with Imaris.

Reply:
I'm not aware, but if there is I'd happily update. I think I've seen RBG in autodesk products